duckduckgo/iOS
# Extract SecureVault code into SecureStorage module

Task/Issue URL: https://app.asana.com/0/0/1205119749390494/f
Tech Design URL:
CC: @Bunn @jotaemepereira 

**Description**:

This PR updates the macOS browser to use the new SecureStorage package.



**Steps to test this PR**:
1. Test that Autofill works as expected (try a clean build, persistence between launches, etc.)



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
